lua/orgmode/config/mappings/map_entry.lua

```diff
@@ -106,6 +106,12 @@ function MapEntry:attach(default_mapping, user_mapping, opts)
     map_opts.prefix = nil
   end
 
+  if user_mapping ~= nil then
+    if user_mapping.desc ~= nil then
+      map_opts.desc = user_mapping.desc
+    end
+  end
```

Comment:
We can remove some nesting here.


```suggestion
  if type(user_mapping) == 'table' and user_mapping.desc then
    map_opts.desc = user_mapping.desc
  end
```